web-ng/src/app/components/map/map.component.ts

```diff
@@ -370,6 +395,35 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  private addPolygonToMap(project: Project, feature: PolygonFeature) {
+    const color = project.layers.get(feature.layerId)?.color;
+    const vertices: google.maps.LatLng[] = [];
+    feature.polygonVertices.map(vertex => {
+      vertices.push(new google.maps.LatLng(vertex.latitude, vertex.longitude));
+    });
+    // Construct the polygon.
```

Comment:
This comment doesn't add any clarity over what's in the code so it can be removed.

Reply:
removed